spec/restaurant_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'restaurant'
+
+describe Restaurant do
+  
+  subject(:restaurant) { described_class.new }
+  
+  it { is_expected.to respond_to(:own_menu)}
+  
+  it 'stores a specific menu' do
+    expect(restaurant.menu).to be_an Hash
+  end
+  
+  describe '#own_menu' do
+  
+  it 'returns its specific menu' do
+    
+    expect(restaurant.own_menu).to eq "Please order from the menu above:"
+   end
+ end
```

Comment:
Lint/BlockAlignment: `end` at 19, 1 is not aligned with `describe '#own_menu' do` at 13, 2.